ayon_api/server_api.py

```diff
@@ -1323,49 +1324,44 @@ def _do_rest_request(self, function, url, **kwargs):
         return new_response
 
     def raw_post(self, entrypoint, **kwargs):
```

Comment:
`entrypoint` == `endpoint` ?

Reply:
Should be `endpoint`, not sure if is related to this PR?